app/controllers/clear_cms/contents_controller.rb

```diff
@@ -126,6 +128,38 @@ def import
       redirect_to({:action=>:index}, notice: 'Content imported successfully.')
     end
 
+    def csv_importer
+    end
+
+    def csv_import
+      headers = CSV.read(params[:dump][:file].path, headers: true).headers
```

Comment:
You are reading the CSV twice...in this case, because it's a small file it's not a big deal, but I would first read it into a var called "rows" and from that get the headers and then cycle through rows (read returns an array of arrays)

http://ruby-doc.org/stdlib-2.2.0.preview1/libdoc/csv/rdoc/CSV.html